### ISSUE TITLE
Is javascript acceptable?

### ISSUE THREAD

**Issue by smishr4** [DivyaGodayal/CoderChef-Kitchen#108]
A lot of programmers already working as frontend developers or some starting their journey in javascript, and struggle to practice ds/algorithms for interview preparation.

Can I submit problems/solutions in js?

**Comment by edorado93** [comment]
Hey @smishr4, any language is fine as long as the article follows the format described on the main README page. Also, make sure to submit an issue for a new problem or an existing one in the repository before you start working on it. Happy Coding!

**Comment by smishr4** [comment]
Alright. I have started covering new problems/solutions on my git, I will push advanced/hard level problems here.

 I have not used markdown, I hope plain text (with screenshot and explaination) is acceptable. I can add jsdoc.

**Comment by edorado93** [comment]
I'm sorry, but the solution file has to be Markdown. Maybe you can convert your text files to Markdown and add the necessary components required for a solution?

**Comment by smishr4** [comment]
plain text meaning description is not markdown formatted. File will be markdown (.md), yes.

**Comment by edorado93** [comment]
Can you post a sample solution file maybe?

**Comment by edorado93** [comment]
Closing this due to inactivity. 
**TLDR**, Solutions in any language are acceptable as long as the article follows the basic template. 
![giphy](https://user-images.githubusercontent.com/11422365/58044217-14bc3b80-7af4-11e9-9201-2a0e110719a7.gif)
